packages/cli/core/src/commands/org/resources/preview.spec.ts

```diff
@@ -159,15 +180,38 @@ describe('org:resources:preview', () => {
   afterEach(() => {
     mockEvaluate.mockClear();
     mockedIsGitInstalled.mockClear();
+    mockedSnapshotFactory.mockClear();
   });
 
-  describe('when the report contains no resources in error', () => {
+  describe('when resources privileges are missing', () => {
     beforeAll(() => {
-      mockSnapshotFactoryReturningValidSnapshot();
+      mockUserNotHavingAllRequiredResourcePrivileges();
     });
 
-    afterAll(() => {
-      mockedSnapshotFactory.mockReset();
+    test
+      .stdout()
+      .stderr()
+      .command(['org:resources:preview'])
+      .catch((ctx) => {
+        expect(formatCliLog(ctx.message, true)).toMatchSnapshot();
```

Comment:
Why do you strip the ANSI esc sequences? It's often colour stuff, so it might be useful to know its been properly colored, no?

Reply:
That's a good point. It was to make the snapshot cleaner. but I guess we want to validate the styling as well